www/install/php/Update-22.10.0-beta.1.php

```diff
@@ -36,6 +36,9 @@
     $errorMessage = "Unable to delete 'appKey' information from database";
     $pearDB->query("DELETE FROM `informations` WHERE `key` = 'appKey'");
 
+    $errorMessage = "Unable to drop 'app_key' from remote_servers table";
+    $pearDB->query("ALTER TABLE remote_servers DROP COLUMN `app_key`");
```

Comment:
column must be checked if exists before

Reply:
Furthermore, must be removed from https://github.com/centreon/centreon/blob/develop/www/install/sql/centreon/Update-DB-22.10.0-beta.1.sql#L126

Reply:
@kduret done :)

Reply:
My second comment is not done :)

Reply:
@kduret ah sorry didn't see the second one sorry :D !